<launch>
  <arg name="anime_mode" default="False" />
  <arg name="rviz" default="False" />
  <node pkg="lqr_control" type="dareSolver.py" name="dare_solver" output="screen">
  </node>
  <node pkg="lqr_control" type="quadrotor_simulator_node" name="quadrotor_simulator" output="screen" >
    <param name="transform_movement_flag" value="true" />
    <param name="verbose" value="true" />
    <param name="anime_mode" value="$(arg anime_mode)" />
    <param name="R_para" value="1.0" />
    <param name="Q_p_mid_para" value="50.0" />
    <param name="Q_v_mid_para" value="0.1" />
    <param name="Q_w_mid_para" value="0.1" />
    <param name="Q_e_mid_para" value="5.0" />
    <param name="Q_z_mid_para" value="50.0" />
    <param name="Q_yaw_mid_para" value="10.0" />
    <param name="start_x" value="0.5" />
    <param name="start_y" value="1.0" />
    <param name="start_z" value="1.2" />
    <param name="start_yaw" value="0.785" />
    <param name="start_time" value="0.0" />
    <param name="end_x" value="1.091" />
    <param name="end_y" value="1.421" />
    <param name="end_z" value="1.14" />
    <param name="end_yaw" value="0.785" />
    <param name="end_time" value="1.0" />
    <param name="mpc_freq" value="20.0" />
    <param name="manual_final_ocp_flag" value="true" />
  </node>
  <group if="$(arg rviz)">
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find aerial_robot_model)/launch/rviz_config" />
  </group>
</launch>
